src/beautifiers/r-beautifier/index.coffee

```diff
@@ -0,0 +1,25 @@
+###
```

Comment:
I believe this file should be removed 😕 ? There is already `src/beautifiers/formatR/index.coffee` I see above.